### PR TITLE
Server README updates for Firebase

### PR DESCRIPTION
**Pull request by burak-turk**
Update to server README to include:
- enabling "Email/Password" sign-in method
- Firebase database creation

### CODE REVIEW

File: server/README.md [hunk 1]
```diff
@@ -33,7 +33,13 @@ For the above example, postgres connection string would be `postgres://omscentra
 
 ## Firebase
 
-If you have not already, create a new project in the [Firebase Console](https://console.firebase.google.com). Once the project is created, click "Add app" from the "Project Overview" page to add a new "Web" app to the project. This Firebase app will handle authentication for your local OMSCentral instance. Finally, go to "Settings" > "Service accounts" tab, click "Generate new private key", and open the generated JSON.
+If you have not already, create a new project in the [Firebase Console](https://console.firebase.google.com). Once the project is created, click "Add app" from the "Project Overview" page to add a new "Web" app to the project. 
+
+This Firebase app will handle authentication for your local OMSCentral instance. Go to "Build" > "Authentication" and click on "Get Started". Under "Sign-in method", enable "Email/Password". 
+
+Next, go to "Build" > "Realtime Database" and click on "Create Database" to create a database for your project.
```
Comment (mehmetbajin) [inline server/README.md:40]: I think we can omit this line b/c omscentral does not use realtime database (it's very expensive!), but there is no harm in leaving it in :)

File: server/README.md [hunk 2]
```diff
@@ -59,7 +65,7 @@ Then, complete `.env` based on the following expectations:
 | OMSCENTRAL_FIREBASE_PRIVATE_KEY  | from firebase private key json                                                                |
 | OMSCENTRAL_FIREBASE_CLIENT_EMAIL | from firebase private key json                                                                |
 | OMSCENTRAL_FIREBASE_PROJECT_ID   | from firebase private key json                                                                |
-| OMSCENTRAL_FIREBASE_DATABASE_URL | from firebase private key json                                                                |
+| OMSCENTRAL_FIREBASE_DATABASE_URL | URL of the database that you created under the Firebase section above                                                               |
```
Comment (mehmetbajin) [inline server/README.md:68]: nit: Could you make sure the rightmost `|` in this line is vertically aligned with those in prior/subsequent lines?

**Review by mehmetbajin — DISMISSED** [review]
LGTM ... just a couple nits. Will merge once the nits are resolved.

File: server/README.md (hunk 1 above)
Comment (mehmetbajin) [reply server/README.md:40]: Discussed offline. The database URL is actually required by the application, so having this line is necessary. Retracting my comment.

File: server/README.md (hunk 2 above)
Comment (burak-turk) [reply server/README.md]: Fixed!

File: server/README.md [hunk 3]
```diff
@@ -59,7 +65,7 @@ Then, complete `.env` based on the following expectations:
 | OMSCENTRAL_FIREBASE_PRIVATE_KEY  | from firebase private key json                                                                |
 | OMSCENTRAL_FIREBASE_CLIENT_EMAIL | from firebase private key json                                                                |
 | OMSCENTRAL_FIREBASE_PROJECT_ID   | from firebase private key json                                                                |
-| OMSCENTRAL_FIREBASE_DATABASE_URL | from firebase private key json                                                                |
+| OMSCENTRAL_FIREBASE_DATABASE_URL | URL of the database that you created under the Firebase section above                         |
```
Comment (mehmetbajin) [inline server/README.md:68]: Nice alignment ... very pretty!

**Review by mehmetbajin — APPROVED** [review]

**Pull request merged**